src/OrchardCore.Modules/OrchardCore.Search.Elasticsearch/ElasticsearchIndexPermissionHelper.cs

```diff
@@ -6,9 +6,10 @@ namespace OrchardCore.Search.Elasticsearch;
 
 public class ElasticsearchIndexPermissionHelper
 {
-    public static readonly Permission ManageElasticIndexes = new("ManageElasticIndexes", "Manage Elasticsearch Indexes");
```

Comment:
I was wondering if there was some kind of circular dependency issue or something else hard to see why this was necessary before, but testing Elasticsearch didn't bring up any issues. So, this just looks like an oversight.

Reply:
I am guessing this was originally placed here incorrectly and kept that way. Everything passes so I don't see any issue from this.